quiz.js

```diff
@@ -36,9 +47,63 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
-
-
+db.houses.update({
+  name: 'House Stark'
+  }, {
+    $push: {
+      members: {
+        name: 'Ned Stark'
+      }
+    }
+  });
+
+db.houses.update({
+  name: 'House Stark'
+  }, {
+    $push: {
+      members: {
+        name: 'Arya Stark'
+      }
+    }
+  });
+
+db.houses.update({
+  name: 'House Stark'
+  }, {
+    $push: {
+      members: {
+        name: 'Sansa Stark'
+      }
+    }
+  });
+
+db.houses.update({
+  name: 'House Targaryen'
+  }, {
+    $push: {
+      members: {
+        name: 'Viserys Targaryen'
+      }
+    }
+  });
+db.houses.update({
+  name: 'House Targaryen'
+  }, {
+    $push: {
+      members: {
+        name: 'Daenerys Targaryen'
+      }
+    }
+  });
+db.houses.update({
+  name: 'House Arryn'
+  }, {
+    $push: {
+      members: {
+        name: 'Jon Arryn'
+      }
+    }
+  });
 
 
```

Comment:
All correct! However, this approach could get tedious if we were trying to add many members; Mongo's [$each](http://docs.mongodb.org/manual/reference/operator/update/each/) operator (shown below) would be a lot DRYer in that kind of scenario.
```
db.houses.update({name: 'House Stark'}, { $push: { members: { $each: 
  [
    { name : 'Ned Stark' },
    { name : 'Arya Stark' }, 
    { name : 'Sansa Stark' }
  ]
}}});
```